pkg/cmd/auth/login/login.go

```diff
@@ -111,6 +114,9 @@ func NewCmdLogin(f *cmdutil.Factory, runF func(*LoginOptions) error) *cobra.Comm
 	cmd.Flags().StringSliceVarP(&opts.Scopes, "scopes", "s", nil, "Additional authentication scopes for gh to have")
 	cmd.Flags().BoolVar(&tokenStdin, "with-token", false, "Read token from standard input")
 	cmd.Flags().BoolVarP(&opts.Web, "web", "w", false, "Open a browser to authenticate")
+	cmd.Flags().BoolVar(&opts.UseGH, "use-gh", false, "Use GitHub.com (don't ask about GitHub Enterprise Server).")
```

Comment:
For consistency, there shouldn't be periods at the end of these long descriptions (since none of the other long descriptions have them)

Reply:
Done.

Reply:
Hmm, strangely the "re-request review" button doesn't work for me.